### PR TITLE
[10.6.X] updating sherpa 2.2.6 to bugfix release sherpa 2.2.7

### PR DESCRIPTION
**Pull request by SiewYan**
As aforementioned [1], sherpa 2.2.7 is the bugfix release.

Attempted on 106X

[1] https://gitlab.com/sherpa-team/sherpa/-/tags/v2.2.7

### CODE REVIEW

File: sherpa.spec [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+%if %isamd64
 Requires: openloops
-%endif # is not ppc64
-%endif # is linux
+%endif # isamd64
+%endif # islinux
 
 %prep
-%setup -q -n SHERPA-MC-%{realversion}
+%setup -q -n SHERPA-MC-2.2.7
```
Comment (smuzaffar) [inline sherpa.spec]: why this change? keep the original one so that when we update version then we do not have to change this.

File: sherpa.spec [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+Source: http://www.hepforge.org/archive/sherpa/SHERPA-MC-2.2.7.tar.gz
 Requires: hepmc lhapdf blackhat sqlite fastjet openssl scons python openmpi
 BuildRequires: mcfm swig
 
 %define islinux %(case $(uname -s) in (Linux) echo 1 ;; (*) echo 0 ;; esac)
 %define isamd64 %(case %{cmsplatf} in (*amd64*) echo 1 ;; (*) echo 0 ;; esac)
 
 %if %islinux
-%ifnarch ppc64le
+%if %isamd64
```
Comment (smuzaffar) [inline sherpa.spec]: is sherpa 2.2.7 not working with openloops on aarch64? I remeber we only had issues with ppc64le archs.

File: sherpa.spec (hunk 1 above)
Comment (SiewYan) [reply sherpa.spec]: Hello, Sorry, i will revert it

**Comment by SiewYan** [comment]
@smuzaffar thanks for the inspection, I have updated a commit, retain only the sherpa version changes.

**Comment by smuzaffar** [comment]
please test
thanks @SiewYan

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Comment by smuzaffar** [comment]
FYI @fabiocos , this is ready to be merged

**Comment by fabiocos** [comment]
please test workflow 534.0,536.0

**Comment by fabiocos** [comment]
@SiewYan @smuzaffar sorry, should not we update the master version before? It seems to me that this is not yet there, please correct me in case

**Comment by smuzaffar** [comment]
@fabiocos , ah sorry I did not notice that it was for 10.6.X , yes we should first get it in for 11.0.X. @SiewYan , please provide a PR for IB/CMSSW_11_0_X/gcc700 please

**Comment by SiewYan** [comment]
@smuzaffar , sorry, i was traveling on monday and tuesday. Thanks for acted on the base branch changes. From my end, do i need to do anything? Thanks

**Comment by smuzaffar** [comment]
@SiewYan , No every thing is in place. By the way, dowe need this in 10.6.X or just in 11.0.X and above should be enough?

**Comment by SiewYan** [comment]
I will need the new sherpa version on 10.6.X for the moment. Could you merge this PR to 10_6_X? Thanks

**Comment by fabiocos** [comment]
@SiewYan normally we first integrate in master, verify everything is ok and then allow for the backport

**Comment by SiewYan** [comment]
@fabiocos , thanks for the clarification. I will open a new PR on 10.6.X backporting for sherpa 2.2.7. Thanks

**Comment by fabiocos** [comment]
@SiewYan well, for 10_6_X this IS the backport :-) @smuzaffar has already prepared the PR for integration in master as far as I can see

**Comment by SiewYan** [comment]
Ah, i am confused by the cascading message.. as @smuzaffar mentioned, i will need to prepare PR for 11_0_X ? thanks

**Comment by SiewYan** [comment]
okies, i have found it:
https://github.com/cms-sw/cmsdist/pull/5196

thanks @fabiocos and @smuzaffar  for the PR. :-)

**Comment by fabiocos** [comment]
@SiewYan sorry, is this request stable? I see further updates. The corresponding master request has been already integrated in master for 11_0_0_pre7, could you please clarify the status?

**Comment by fabiocos** [comment]
please test workflow 534.0,536.0

**Comment by SiewYan** [comment]
@fabiocos , the sherpa update is stable. I intended an update on openloops on different PR https://github.com/cms-sw/cmsdist/pull/5204 (110X) https://github.com/cms-sw/cmsdist/pull/5205 (106X) for book keeping.

For some reason the initial attempt on integrating openloop committed on this branch... I apolagize. I am try to revert the commits 4a63fd4 , 7ca9a89  , f9232c2 at the meantime...
I have no idea how to drop those commits, could you help me with that? thanks ><

**Comment by smuzaffar** [comment]
+externals

**Comment by fabiocos** [comment]
@SiewYan please replace this PR with clean backports of the master versions

**Comment by smuzaffar** [comment]
-externals
I am closing this and will make a proper back port of #5196
